tools/test/src/tools/EmulatorHelper.scala

```diff
@@ -20,6 +20,7 @@ object EmulatorHelper {
   def test(
       modelName: String,
       inputBatchSize: Int = 1,
+      localConsts: Boolean = false,
```

Comment:
how are we checking that vars are also stored locally in the `local-vars` tests?

Reply:
Emulator needs  `localConsts` flag to load consts either to DRAM1 or to local memory. When vars are passed through local memory it's captured by the program itself, so emulator does not need to know this for the test to assert output correctness.